_posts/2020-02-10-newpipe-0.18.3-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.3 released: Hotfix for the SoundCloud Extractor"
+short: "NewPipe 0.18.3 released"
+date: 2020-02-10 00:00:00 +0100
+categories: [release, pinned]
+author: Stypox
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A change on SoundCloud side made it necessary to make some edits to the SoundCloud Extractor, so a new update for NewPipe is here!
+
+## Hotfix
+
+Since this is a hotfix release, it does not contain any new feature or bugfix, except for the SoundCloud Extractor fixup. If you see any Guru Mediation error in the app, please update to 0.18.3 before you create an issue (you can verify if you actually have `0.18.3+` installed by tapping `About` in the app drawer menu).
+
+<!-- more -->
+
+SoundCloud changed the API key (called `client_id`), as it uses to do rouughly once every 1-2 months. The app's hardcoded client id was updated by [@TobiGr](https://github.com/TobiGr). Further investigation is required to understand why the automatic `client_id` extraction failed, after the hardcoded id became invalid.
```

Comment:
`rooughly` should be `roughly`. `ID` should be capitalized. There should be no comma after `failed`.